.github/workflows/presubmit.yml

```
@@ -452,3 +452,62 @@ jobs:
       - name: Ninja log
         run: |
           cat build/.ninja_log
+      - name: Produce install artifacts
+        run: |
+          cmake --install build --prefix build/install_artifact --config Release
```

nit: since we're not using the multi-config genreator, do we need --config? Also, this is a debug build
```suggestion
          cmake --install build --prefix build/install_artifact
```

---

Need to be updated according to #679